### DISCUSSION TITLE
Running CARACAL on a shared computing cluster with slurm

### DISCUSSION THREAD

**Discussion by ycp21** [caracal-pipeline/caracal#1473] [Q&A] [answered]
Hi everyone, I'm just getting started with CARACAL to process some MeerKAT data (L-band continuum, ~3 hour observations).  I'm running on a shared computing cluster so I need to have some estimate of time, CPU, memory requirements to request jobs and I can't see anything about this in the documentation.  I know it'll be a bit trial-and-error, but does anyone have any experience and rules of thumb to help with initial guesses?  Thanks for any help!

**Comment by paoloserra** [comment, marked as answer]
Sect. 3 of https://ui.adsabs.harvard.edu/abs/2023arXiv230211895S describes what worked for me. It's probably not exactly the type of data you're working on, but it might be of some guidance.

**Reply by ycp21** [reply]
Thanks Paolo, this is a very helpful starting point!
